README.md

```diff
@@ -1 +1,50 @@
-# ramen
+# Ramen
+
+Ramen is a **disaster-recovery orchestrator** for stateful applications across
+a set of peer kubernetes clusters that,
+
+- Preserve state using persistent volumes provisioned using [ceph-csi](https://github.com/ceph/ceph-csi/)
+- Uses the CSI [storage replication extensions](https://github.com/csi-addons/volume-replication-operator)
+  , which enable ceph-csi to replicate state across peer clusters
+- Are deployed and managed using [open-cluster-management](https://open-cluster-management.io/concepts/architecture/)
+
+Ramen provides **cloud-native interfaces** to orchestrate the life-cycle of an
+applications state across peer clusters. These include,
```

Comment:
"an application's state" instead of "an applications state"?  In multiple places below.